zeppelin-server/src/test/java/org/apache/zeppelin/rest/ZeppelinSparkClusterTest.java

```diff
@@ -136,23 +145,25 @@ public void sparkSQLTest() throws IOException {
             p.setAuthenticationInfo(anonymous);
             note.run(p.getId());
             waitForFinish(p);
-            assertEquals(Status.FINISHED, p.getStatus());
-            assertTrue(p.getResult().message().get(0).getData().contains(
-                    "Array[org.apache.spark.sql.Row] = Array([hello,20])"));
+            assertThat(p.getStatus(), is(Status.FINISHED));
+            assertThat(p.getResult().message(), contains(
+                hasProperty("data", containsString("Array[org.apache.spark.sql.Row] = Array([hello,20])"))
+            ));
 
             // test display DataFrame
             p = note.addNewParagraph(AuthenticationInfo.ANONYMOUS);
             config = p.getConfig();
             config.put("enabled", true);
             p.setConfig(config);
-            p.setText("%spark val df=sqlContext.createDataFrame(Seq((\"hello\",20)))\n" +
-                    "z.show(df)");
+            p.setText("%spark val df=sqlContext.createDataFrame(Seq((\"hello\",20)))\nz.show(df)");
             p.setAuthenticationInfo(anonymous);
             note.run(p.getId());
             waitForFinish(p);
-            assertEquals(Status.FINISHED, p.getStatus());
-            assertEquals(InterpreterResult.Type.TABLE, p.getResult().message().get(1).getType());
-            assertEquals("_1\t_2\nhello\t20\n", p.getResult().message().get(1).getData());
+            assertThat(p.getStatus(), is(Status.FINISHED));
```

Comment:
This seems some kind of code style change, it may not be necessary, because I think we should reach consensus among community to make our code style consistent. Open source project also require code style as consistent as possible